metadata.json

```diff
@@ -65,7 +65,6 @@
     }
   ],
   "dependencies": [
-    {"name":"ghoneycutt/sysklogd","version_requirement":">= 1.0.0 < 2.0.0"},
```

Comment:
It is still a soft dependency and this is needed for the spec tests to work so it should stay. Also doesn't hurt anything by it being there, so I don't understand the motivation to remove it.

Reply:
Can you please remove the change to `metadata.json` from your pull request and squash/rebase. The other change to only require sysklodg inside the EL5 case statement is still needed.

Reply:
Done (I think, I'm not used to github)